mods/d2k/chrome.yaml

```diff
@@ -100,6 +100,7 @@ flags: buttons.png
 	smuggler: 111,114,23,21
 	fremen: 133,114,23,21
 	worm: 156,114,23,21
+	mercenary: 179,114,20,21
```

Comment:
Can you please fix it so that all the flags are 22x21?  The corrino, fremen, and worm flags are all 22x21, not 23x21.  Harkonnen and Random are the odd ones out at 23x21.